src/platform/K32W/k32w-chip-mbedtls-config.h

```diff
@@ -1,6 +1,5 @@
 /*
- *   Copyright (c) 2020 Project CHIP Authors
- *  Copyright (c) 2020, The OpenThread Authors.
+ *  Copyright (c) 2019, The OpenThread Authors.
```

Comment:
Add back copyright?

Also this sems to lose some changes (like pragma once)

Reply:
Added the correct copyright - 2021 Project CHIP Authors.